pkgs/applications/networking/instant-messengers/webcord/default.nix

```diff
@@ -6,23 +6,23 @@
 , pipewire
 , libpulseaudio
 , xdg-utils
-, electron_25
+, electron
```

Comment:
Has the electron 26 crash bug been fixed, and we can switch back to 26?

Also, I'd prefer to specify the electron version specifically, just to avoid electron updates in nixpkgs breaking the package.

Reply:
`electron` is `electron_27`

i can specify electron_27 yes

Reply:
Oh cool 27 is out didnt know that, hopefully the end of weird electron crashes is nigh